llm_mining_core/utils/requests_utils.py

```diff
@@ -42,7 +42,8 @@ def send_miner_request(config, miner_id, model_id):
     request_data = {
         "miner_id": miner_id,
         "model_id": model_id,
-        "min_deadline": 1
+        "min_deadline": 1,
+        "accept_stream": True
```

Comment:
please revert this - default to not accepting streaming

Reply:
rm'ed